tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -325,4 +332,244 @@ public function testListFloatTypeColumns()
         self::assertFalse($columns['col']->getUnsigned());
         self::assertTrue($columns['col_unsigned']->getUnsigned());
     }
+
+    /**
+     * As of MariaDB 10.2.7, nullable default values literals are always single quoted in
+     * information_schema. Non-nullable defaults behaviour is not affected.
+     * This test ensure accidental removal of double single encoded defaults for MariaDB >= 10.2.7.
+     *
+     * @link https://mariadb.com/kb/en/library/information-schema-columns-table/
+     * @link https://dev.mysql.com/doc/refman/5.5/en/string-literals.html
+     */
+    public function testColumnDefaultValuesDoubleQuoted() : void
```

Comment:
This test should be moved to a generic `SchemaManagerTest` which doesn't yet exist. All the assertions below are valid for all supported platforms.

Reply:
No answers for that, I'll summarize when I've fixed reviews.